lib/vhdl-linter.ts

```diff
@@ -778,7 +812,13 @@ export class VhdlLinter {
         addTypes(iterator.types, 500);
       }
     }
-    return subprograms.filter(e => e.lexerToken.getLText() === instantiation.componentName.text.toLowerCase());
+    // in entities
+    // subprograms.push(...this.projectParser.entities.flatMap(ent => ent.subprograms));
```

Comment:
```suggestion

```

Reply:
Yeah, I dont know what that's supposed to do, the "in entities" does not explain anything.
Commenting out does not break any testcase, but yeah removing is better.